### PR TITLE
React rails

### PR DESCRIPTION
**Pull request by mirterious**
# PR Details

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: app/javascript/components/HelloWorld.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import PropTypes from "prop-types"
+class HelloWorld extends React.Component {
+  render () {
+    return (
+      <React.Fragment>
+        Greeting: {this.props.greeting}
+        <p>Пиписька</p>
```
Comment (nieisnuje) [inline app/javascript/components/HelloWorld.js]: Remove this please

File: app/assets/fonts/futura/sharefonts.net.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Free download fonts at http://sharefonts.net
```
Comment (epfeen) [inline app/assets/fonts/futura/sharefonts.net.txt]: delete this.

File: app/assets/stylesheets/homepage.scss [hunk 3]
```diff
@@ -0,0 +1,40 @@
+// Place all the styles related to the Homepage controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+$black: #1b1b1b;
+$gray: #e4e4e4;
+$orange: #ff6d51;
+
+//  фон
```
Comment (epfeen) [inline app/assets/stylesheets/homepage.scss]: english.

File: app/assets/stylesheets/start.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
+// Place all the styles related to the Homepage controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+$black: #1b1b1b;
+$gray: #e4e4e4;
+$orange: #ff6d51;
+
+//  фон
```
Comment (epfeen) [inline app/assets/stylesheets/start.scss]: english.

File: app/javascript/components/HelloWorld.js (hunk 1 above)
Comment (epfeen) [inline app/javascript/components/HelloWorld.js]: pipiska.

Comment (nieisnuje) [reply app/javascript/components/HelloWorld.js]: Lets make it h1

File: spec/helpers/dictionary_helper_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+# Specs in this file have access to a helper object that includes
+# the DictionaryHelper. For example:
+#
+# describe DictionaryHelper do
```
Comment (epfeen) [inline spec/helpers/dictionary_helper_spec.rb]: waaaaaaat???

File: spec/helpers/homepage_helper_spec.rb [hunk 6]
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+# Specs in this file have access to a helper object that includes
+# the HomepageHelper. For example:
+#
+# describe HomepageHelper do
```
Comment (epfeen) [inline spec/helpers/homepage_helper_spec.rb]: waaaaaat(2)?

File: spec/helpers/manifesto_helper_spec.rb [hunk 7]
```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+# Specs in this file have access to a helper object that includes
+# the ManifestoHelper. For example:
+#
+# describe ManifestoHelper do
```
Comment (epfeen) [inline spec/helpers/manifesto_helper_spec.rb]: waaaaaat(3)?

File: app/assets/stylesheets/homepage.scss [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+//  кнопка
+.btn-secondary {
+  background-color: $orange;
+  border-color: $orange;
+  color: $gray;
+}
+// хедер
+.jepa {
```
Comment (epfeen) [inline app/assets/stylesheets/homepage.scss]: rename.

**Pull request merged**